Userland/Utilities/lscpu.cpp

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2022, Undefine <undefine@undefine.pl>.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/Format.h>
+#include <AK/JsonArray.h>
+#include <AK/JsonObject.h>
+#include <LibCore/File.h>
+#include <LibCore/System.h>
+#include <LibMain/Main.h>
+
+static void parse_processor(JsonValue const& value)
+{
+    auto const& processor = value.as_object();
+
+    outln("CPU {}:", processor.get("processor").to_string());
+    outln("\tVendor ID: {}", processor.get("cpuid").to_string());
+    outln("\tModel: {}", processor.get("brand").to_string());
+    outln("\tFamily: {}", processor.get("family").to_string());
+    outln("\tModel: {}", processor.get("model").to_string());
+    outln("\tStepping: {}", processor.get("stepping").to_string());
+    outln("\tType: {}", processor.get("type").to_string());
+    out("\tFeatures: ");
+
+    for (size_t i = 0; i < processor.get("features").as_array().size(); i++) {
+        out("{} ", processor.get("features").as_array()[i].to_string());
```

Comment:
That's not what I meant. You can still store it in a separate variable, but make it a reference instead of a copy (`auto& foo = ...` vs `auto foo = ...`)